pilot/pkg/config/clusterregistry/secretcontroller.go

```diff
@@ -127,24 +122,6 @@ func (c *Controller) Run(stopCh <-chan struct{}) {
 	wait.Until(c.runWorker, 5*time.Second, stopCh)
 }
 
-// StartSecretController start k8s controller which will be watching Secret object
-// in a specified namesapce
-func StartSecretController(k8s kubernetes.Interface,
-	cs *ClusterStore,
-	serviceController *aggregate.Controller,
-	discoveryServer *envoy.DiscoveryServer,
-	namespace string,
-	resyncInterval time.Duration,
-	watchedNamespace,
-	domainSufix string) error {
-	stopCh := make(chan struct{})
```

Comment:
For reviewers: this stop channel should be that from discoveryCmd. So I change this to use `Server.addStartFunc` to pass stop channel.